src/PacketaWidget/useOnWidgetButtonClicked.js

```diff
@@ -4,12 +4,14 @@
  * @package Packetery
  */
 
-import { useCallback, useState } from 'react';
+import { useCallback } from 'react';
 
 export const useOnWidgetButtonClicked = (
 	packetaShippingRate,
 	settings,
-	dynamicSettings
+	dynamicSettings,
+	viewState,
```

Comment:
Tohle se tam koukám nikde nepoužívá, jen se to vracelo. Takže to asi není potřeba sem ani předávat.